android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java

```diff
@@ -120,7 +119,9 @@ public void onBindViewHolder(RecyclerViewHolder holder, int position) {
                     + feedItem.getLink() + "</a> "));
 
             // Making url clickable
-            holder.url.setMovementMethod(LinkMovementMethod.getInstance());
+            holder.url.setOnClickListener(v -> {
+                Utils.setUpCustomTab(context, feedItem.getLink());
+            });
```

Comment:
Please rename lambda variable to `view`

And in case of one line functions, it's ok to remove the curly braces for brevity :smile: